src/redux/weather.js

```diff
@@ -0,0 +1,41 @@
+import { createAction, createAsyncThunk, createReducer } from '@reduxjs/toolkit';
+import fetchSingleCity from '../services/connectAPI';
+
+const SELECT_CITY = 'SELECT_CITY';
+const FETCH_CITY_WEATHERS = 'FETCH_CITY_WEATHERS';
+
+const cities = ['Moscow', 'London', 'Paris', 'Cairo', 'Dubai', 'Kigali', 'Madrid', 'New York', 'Roma', 'Nairobi', 'Dodoma', 'Ottawa'];
+
```

Comment:
[OPTIONAL] 👍🏽 
- I believe it will be amazing if I can view any city at all in the world. Someone may randomly want to see how your mobile app works by checking their own city. Just as @cynthiainga suggested, It will be awesome to use an API to fetch data for all nations and cities in different continents. 🌸